projects/plugins/jetpack/extensions/blocks/donations/view.js

```diff
@@ -110,7 +114,7 @@ class JetpackDonations {
 
 	initNavigation() {
 		const navItems = this.block.querySelectorAll( '.donations__nav-item' );
-		const tabContent = this.block.querySelector( '.donations__tab' );
+		const tabContent = this.block.querySelector( '.donations__content' );
```

Comment:
Hmmm I'm unable to reproduce it. Are you on an Atomic site? Maybe you are saving before the Donations Block actually has fetched all the information it needs. The donations block is doing some API calls in the background but shows an editable placeholder in the meantime.

Reply:
Yeah that's probably it.

Reply:
I'm thinking about creating a separate issue for this so that we can discuss more in detail what we expect to happen in this case. Maybe we could show a skeleton/loader while the content loads or the odds of a user reproducing this during a normal workflow is so small that we can live it. But I think it's going to require more consensus than what we can discuss during a PR review.

Reply:
This should also fix this issue https://github.com/Automattic/jetpack/issues/22838